client/src/main/kotlin/io/spine/examples/pingh/client/TimestampExts.kt

```diff
@@ -64,15 +65,18 @@ internal fun Timestamp.add(duration: com.google.protobuf.Duration): Timestamp =
  * - If the difference is exactly one day, returns the string `"yesterday"`;
  * - In all other cases, returns the day and month.
  *
+ * @receiver The timestamp represents a time in the past.
  * @throws IllegalArgumentException if the `Timestamp` is not from the past.
  */
 public fun Timestamp.howMuchTimeHasPassed(): String {
-    val thisDatetime = LocalDateTime.ofEpochSecond(this.seconds, this.nanos, ZoneOffset.UTC)
+    val thisDatetime = this.asLocalDateTime()
     val difference = Duration.between(
         thisDatetime,
-        LocalDateTime.now(ZoneOffset.UTC)
+        LocalDateTime.now()
     )
-    require(difference > Duration.ZERO) { "" }
+    require(difference > Duration.ZERO) {
+        "This `Timestamp` must indicate a time in the past."
```

Comment:
With the error messages like this popping out in the middle of some production runtime, it would be very-very helpful to give more information for debugging. Right now we give nothing :) Let's print out the values that caused the story.